examples/durable-objects-websocket/src/routes/index.tsx

```diff
@@ -17,7 +17,7 @@ export function routeData() {
 }
 
 export default function Home() {
-  const user = useRouteData<ReturnType<typeof routeData>>();
+  const user = useRouteData<typeof routeData>();
```

Comment:
The ReturnType here causes lack of types for user

Reply:
Yeah we can remove it